components/scream/data/scream_input.yaml

```diff
@@ -16,4 +16,5 @@ Atmosphere Driver:
     Reference Grid: SE Physics
```

Comment:
Is SE Physics deprecated now as a reference grid?  I get the following error when unit testing:
```
  /p/lustre2/donahue5/E3SM_simulations/SCREAM/scream_development/
  rain_evap_tests/scream/components/scream/src/physics/share/
  physics_only_grids_manager.cpp:33: FAIL:
  grid_name=="Physics Only" || grid_name=="Physics"
  Error! Only 'Physics Only' (and, for convenience, 'Physics') grid supported
  for physics-only runs.
         Requested grid: SE Physics
```

which goes away if I delete SE.

Reply:
The "Physics Only" grid manager does not produce a SEGrid, which is why "SE Physics" won't work. I will fix the ad input file.